dhanhq/marketfeed.py

```diff
@@ -31,6 +31,10 @@
 Quote = 17
 Depth = 19
 
+"""Constants for Unsubsribe Code"""
```

Comment:
"Unsubscribe" spelling to be fixed
and simple nomenclature to unsubscribe like: `Un_Ticker` or `Un_Quote`.